appserver/static/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
```

Comment:
Why is this file in both the static and template folders? 
Also, this flask documentation seems to say that html should mostly go in the templates folder while the static folder mostly holds css & js files:
https://flask.palletsprojects.com/en/1.1.x/tutorial/static/
https://flask.palletsprojects.com/en/1.1.x/tutorial/templates/